### ISSUE TITLE
In "Show POIs long the route" option, allow user to delete the points in bulk

### ISSUE THREAD

**Issue by raindropsfromsky** [osmandapp/Osmand#3792]
Usually when we want to look for a POI of certain type along the route, we want to visit a single point.

For example, a gas station. After picking the most convenient gas station, we want to delete the rest.

But the OsmAnd GUI is not meant for bulk-removal of these POIs: You have to tap on the X button of each POI one by one.

This is tedious.

**Desired:** Provide a bulk-selection mode (like how search history entries can be selected in bulk and deleted).

**Comment by vshcherb** [comment]
Today, you could add intermediate destination from that point and disable others. In that case you won't miss that point cause the navigation guide you through.

**Comment by raindropsfromsky** [comment]
Yes, but this workaround is for experts only. It won't occur to most users.
On the other hand, the following is intuitive:
1. User long-presses on any POI. OsmAnd shows checkboxes against all POIs, 
2. OsmAnd also shows a master checkbox at top. Selecting this box selects all. </br> Unselecting this box unselects all POIs together.
3. After selecting the entries in bulk, the user taps on a few POIs to toggle their selection status.
4. User taps on delete button to delete all unwanted POIs in one stroke.
